pkg/obfuscator/keywords.go

```diff
@@ -24,8 +20,8 @@ func (o *keywordsObfuscator) Contents(contents string) string {
 func replace(name string, replacements map[string]string, reporter ReplacementTracker) string {
 	for keyword, replacement := range replacements {
 		if strings.Contains(name, keyword) {
+			reporter.AddReplacementCount(keyword, keyword, replacement, uint(strings.Count(name, keyword)))
 			name = strings.Replace(name, keyword, replacement, -1)
```

Comment:
silly question, if we're already doing the counter work, can't we reuse that for the replace call?

so keep the 
> cnt := strings.Count(name, keyword)

and then pass it along the replace:

> name = strings.Replace(name, keyword, replacement, cnt)

Reply:
Good suggestion, but `strings.Replace()` calls `Count` internally as well. Even if we pass `count` to `Replace`, it will be ignored https://github.com/golang/go/blob/master/src/strings/strings.go#L948

Reply:
liked the stuff you added on slack, I think we can save ourselves the contains check entirely then.
```
if count := strings.Count(name, keyword); count > 0 { 
  reporter.AddReplacementCount(keyword, keyword, replacement, uint(count))
  name = strings.Replace(name, keyword, replacement, -1)
}
```

Reply:
I wouldn't do that. `Index()` is potentially very CPU heavy operation. `Contains()` calls `Index()` once, `Count()` calls `Index()` in a for loop.

Now we call `Count()` only when `Contains() == true`, thus works as a flood gate before we call `Count()`